developer-docs-site/docs/sdks/unity-sdk.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Unity SDK"
+slug: "unity-sdk"
+---
+
+
+# Aptos Unity SDK
+
+The [Aptos Unity SDK](https://github.com/aptos-labs/Aptos-Unity-SDK) is a .NET implementation of the [Aptos SDK](./index.md), compatible with .NET Standard 2.0 and .NET 4.x for Unity. The goal of this SDK is to provide a set of tools for developers to build multi-platform applications (mobile, desktop, web, VR) using the Unity game engine and the Aptos blockchain infrastructure.
+
+See the post [Aptos Labs brings Web3 to Gaming with its new SDK for Unity developers](https://medium.com/aptoslabs/aptos-labs-brings-web3-to-gaming-with-its-new-sdk-for-unity-developers-e6544bdf9ba9) for all of the features offered to game developers by the Aptos Unity SDK.
+
+## Core features
+
+
+- HD wallet creation and recovery
+- Account management
+    - Account recovery
+    - Message signing
+    - Message verification
+    - Transaction management
+    - Single / Multi-signer Authentication
+    - Authentication key rotation
+- Native Binary Canonical Serialization (BCS) support
+- Faucet client for devnet
+
```

Comment:
FYI this is a duplicate of https://github.com/aptos-labs/Aptos-Unity-SDK#technical-details

Maybe we just link to that, otherwise, we'll just have to tell anyone who updates the SDK to update this page as well

Reply:
Done in the next commit. Thanks!